Formula/s/symfony-cli.rb

```diff
@@ -0,0 +1,19 @@
+class SymfonyCli < Formula
+  desc "Build, run, and manage Symfony applications"
+  homepage "https://github.com/symfony-cli/symfony-cli"
+  url "https://github.com/symfony-cli/symfony-cli/archive/refs/tags/v5.10.9.tar.gz"
+  sha256 "1b08e646f8127436deeb7dab910248a061381e7a8e742c34e713d96d0ee30e3a"
+  license "AGPL-3.0-only"
```

Comment:
```suggestion
  license "AGPL-3.0-or-later"
```
https://github.com/search?q=repo%3Asymfony-cli%2Fsymfony-cli+%22or+%28at+your+option%29%22&type=code